support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -70,11 +83,15 @@ switch (countryGroupId) {
 		break;
 }
 
+const isSignedIn = !!get('GU_U');
+const countryId: IsoCountry =
+	CountryHelper.fromString(get('GU_country') ?? 'GB') ?? 'GB';
```

Comment:
I thought we might be able to infer this from `countryGroupId` but we need the actual country in the `StateSelect`.